ui.apps/src/main/content/jcr_root/apps/cq-bootcamp/component/Content/body/banner/banner.jsp

```diff
@@ -0,0 +1,27 @@
+<%@ page import="sun.net.NetProperties" %>
+<%@include file="/apps/cq-bootcamp/global.jsp" %>
+<%@page session="false" %>
+
+<div class="cardRow hpCsrBg"
+     style='background-image: url(${properties.fileReference}); padding: 10px 10px 257px 10px'>
+    <div class="col">
+        <div class="sliderWrapper">
+            <cq:text value="${properties.title}"  tagName="h2" default="Corporate Social Responsibilities" />
```

Comment:
there is a better way to use cq:text. <cq:text property="title" tagame=... default=''' placeholder=''/>